wger/gym/views/gym.py

```diff
@@ -172,6 +174,57 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class GymMemberComparisonView(LoginRequiredMixin, WgerMultiplePermissionRequiredMixin, ListView):
+    """Overview of gym member comparison for a specific gym."""
+
+    model = User
+    permission_required = ('gym.manage_gym', 'gym.gym_trainer', 'gym.manage_gyms')
+    template_name = 'gym/member_comparison.html'
+
+    def dispatch(self, request, *args, **kwargs):
+        """Only managers and trainers for this gym can access the members."""
+        if request.user.has_perm('gym.manage_gyms') or\
+            ((request.user.has_perm('gym.manage_gym') or
+              request.user.has_perm('gym.gym_trainer')) and
+                request.user.userprofile.gym_id == int(self.kwargs['pk'])):
+            return super(GymMemberComparisonView, self).dispatch(request, *args, **kwargs)
+        return HttpResponseForbidden()
+
+    def get_queryset(self):
+        """Return a list with the users, not really a queryset."""
+        members = {'members': [],
+                   'users': []}
+        for member in UserProfile.objects.all():
+            members['members'].append(member)
+
+        for u in Gym.objects.get_members(self.kwargs['pk']).select_related('usercache'):
+            members['users'].append({'obj': u})
+        return members
+
+    def get_context_data(self, **kwargs):
+        """Pass other info to the template."""
+        # out2 = UserProfile.objects.filter(user_id=10).all()
+        context = super(GymMemberComparisonView, self).get_context_data(**kwargs)
+        context['gym'] = Gym.objects.get(pk=self.kwargs['pk'])
+        context['name'] = "Members Comparison"
+        context['user_table'] = {'keys': [_('ID'), _('Username'), _('Select')]}
+        context['members'] = {'members': context['object_list']['members'],
+                              'users': context['object_list']['users']}
+        data = {}
+        for member in context['members']['members']:
+            if member.calories:
```

Comment:
This can be wrapped into a ternary operation [reference](http://book.pythontips.com/en/latest/ternary_operators.html)